src/crossfilter.js

```diff
@@ -121,6 +138,11 @@ function crossfilter() {
     offset = tmp.offset;
     one = tmp.one;
     zero = ~one;
+      
+    // Expose the dimension mask for internal use
+    dimension._offset = offset;
```

Comment:
I would be against using indexOf to look up the entry, since that can be a performance concern and we wouldn't want that on a performance critical path.

(Ignore my other comment on exposing the map at the crossfilter scope, I didn't realize the dimension was in the crossfilter closure lexical scope.)

Therefore, exposing an id or key property for the dimension could work.  I could use that to look up in a map.  Or, actually, just programatically compute the mask based on the ID.  This would avoid the need for the map lookup.  And I could just set the id property in the dimension construction based on the provided offset/one values.